app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Do you think we should federate this `subscribed` info? What user see in his stream feels like a local setting, rather than network-wise. So `sharing`/`receiving` is a representation of exchange relationships in federation, while `following` is just a setting of composing the stream. We can include `following` since user will want to preserve this through migration, while `subsribed` is something meaningful only for the remote person, not for the local one. Right?